### DISCUSSION TITLE
Ubuntu 22.04 and OpenVINO Notebooks

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#588] [Show and tell]
Ubuntu 22.04 comes with Python 3.10 by default. The easiest workaround to OpenVINO installed properly on 22.04 today is to add Python 3.9.

https://linuxhint.com/install-python-ubuntu-22-04/

It would be a simple few steps and as long as you start your virtual environment from python3.9, you should be set!

That's all ;)

**Comment by nilutpolkashyap** [comment]
Hi @raymondlo84 
After installing Python 3.9, can I install Openvino using pip in the virtual environment or do I need to build it from the source?

Thanks

**Reply by raymondlo84** [reply]
pip install is all you need. ;) We will support 3.10 officially in the next release as well.
